### PR TITLE
Massive board fix

### PR DESCRIPTION
**Pull request by katemorris**
This is HUGE. I got it working! But there were bugs. One bug remains which is human place ships does not know that AA and AB are consecutive. However, there is logic for the computer to find consecutive placements so maybe we can use that.

### CODE REVIEW

File: lib/player.rb [hunk 1]
```diff
@@ -168,14 +208,38 @@ def find_cells_hit
     end
   end
 
+  def generate_adjactent_letters(coord)
+    ord_letter_array = coord_letter(coord).split('').map { |letter| letter.ord }
+    same_double = ord_letter_array.map { |ord| ord.chr }.join
```
Comment (carson-jardine) [inline lib/player.rb:213]: Could we use coord here?

File: lib/player.rb [hunk 2]
```diff
@@ -168,14 +208,38 @@ def find_cells_hit
...
-      split_coord = coord.split('')
-      ord_letter = split_coord.first.ord
-      int_num = split_coord.last(split_coord.count-1).join.to_i
-      letters = [ord_letter.chr, ord_letter.chr, (ord_letter - 1).chr, (ord_letter + 1).chr]
-      numbers = [(int_num + 1).to_s, (int_num - 1).to_s, split_coord[1], split_coord[1]]
-      letters.zip(numbers).map { |coords| coords.join }
+      int_num = coord_num(coord)
+      numbers = [(int_num + 1).to_s, (int_num - 1).to_s, int_num, int_num]
+      dashes = ["-", "-", "-", "-"]
+      generate_adjactent_letters(coord).zip(dashes, numbers).map { |coords| coords.join }
```
Comment (carson-jardine) [inline lib/player.rb:242]: ZIPPP!!!!

**Review by carson-jardine — COMMENTED** [review]
Even though this is long af, it is awesome and makes sense to me!! Definitely going to take a look at refactoring but WOW

File: lib/player.rb (hunk 1 above)
Comment (katemorris) [reply lib/player.rb:213]: SOLID idea!  but alas, it returns an extra number, broke a test. Tried it and reverted.

**Pull request merged**